aloha-core/src/main/scala/com/eharmony/aloha/dataset/LabelRowCreator.scala

```diff
@@ -2,6 +2,6 @@ package com.eharmony.aloha.dataset
 
 import com.eharmony.aloha.semantics.func.GenAggFunc
 
-trait LabelRowCreator[-A] extends RowCreator[A] {
+trait LabelRowCreator[-A, +B] extends RowCreator[A, B] {
```

Comment:
```scala
trait LabelRowCreator[-A] extends CharSeqRowCreator[A]
```

Reply:
**NO** @ryan-deak-zefr!  *This is not a good idea.*

Reply:
I think you need to take a break @ryan-deak-zefr.  @deaktator get back to work!!